hack/test-cmd.sh

```diff
@@ -238,6 +238,9 @@ fi
 PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
 
 kube::log::status "Checking kubectl version"
+
+# Note that this fails on a dirty commit.  To avoid that from happening, .gitignore
```

Comment:
Hmmmm... It was happening in the ci, and the fix was to do git ignore .

I think iIt's specific to ci ?